### DISCUSSION TITLE
idea: trigger an Error/Warning when a loop is set but the variable not used

### DISCUSSION THREAD

**Discussion by bmillemathias** [ansible/ansible-lint#1317] [Ideas]
##### Summary
I did something like this
```
- debug:
    msg: "foobar"
  with_items:
    - one
    - two
    - three
```

do you think that would bring value to return a error is the variable "{{ item }}" is not used ?

##### Issue Type

- Feature Idea

##### Additional Information

```console

```

**Comment by netzvieh** [comment]
I like the idea, but we'd have to exclude `include_*` from this, as the variable could be used inside the included yml.
